infra/local/locals.inc.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+function generate_locals {
+  local SCRIPT_DIR PGPORT PGPASS PGPASS_FILE PSQL_FILE
+
+  SCRIPT_DIR="$1"
+  if [[ ! -d "$SCRIPT_DIR" ]]; then
+    printf "generate_locals missing arguments (script_dir)\n" 1>&2
+    return 1
+  fi
+
+  PGPASS_FILE="$SCRIPT_DIR/pgpass.local.conf"
+  PSQL_FILE="$SCRIPT_DIR/psql.local.sh"
+
+  PGPORT=$(docker port bytecodealliance-registry-db-1 5432 | sed -e 's/.*://g')
```

Comment:
A. Did this change in a recent commit? I thought `PGPASS` was randomized.
B. Can you add a comment that documents what this section does, links any references for the pattern, and comments on the security?

Reply:
A. Yes, I backtracked on reading sensitive information from files since I was thinking of doing it in a follow up PR. Instead I've just done it as properly as I think I can at the moment with the latest additional patches.
B. In the latest patches I added and/or fixed the doc comment at the top of the file.